milli/src/update/index_documents/typed_chunk.rs

```diff
@@ -177,6 +180,24 @@ pub(crate) fn write_typed_chunk_into_index(
             )?;
             is_merged_database = true;
         }
+        TypedChunk::GeoPoints(mut geo_points) => {
+            let mut rtree = index.geo_rtree(wtxn)?.unwrap_or_default();
```

Comment:
Do we have an idea of the rtree size? I'm a bit worried about smaller VMs, this rtree will grow on each document indexing; and because it's an uncontrolled memory usage, it could make meilisearch be killed.

Reply:
Yes, this will be a problem :pensive: 
Currently, for 1 million documents with a geopoint it looks like the `rtree` consume around 400Mb

Reply:
OUCH! @curquiza we may add a warning in the documentation 😬, some indexing could crash because of it.

Reply:
I'm not sure, but I think @gmourier told the doc team about that and asked to put a big `WARNING: BETA` flag on top of the documentation or something like that? :thinking: